src/AppBundle/DataFixtures/ORM/LoadRequestTypeData.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace AppBundle\DataFixtures;
+
+use Doctrine\Common\DataFixtures\AbstractFixture;
+use Doctrine\Common\DataFixtures\OrderedFixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use AppBundle\Entity\RequestType;
+
+
+class LoadRequestTypeData extends AbstractFixture implements OrderedFixtureInterface
```

Comment:
@lilia86 Why you not using Alice yaml fixtures? It's much easier to read and modify